actors/builtin/power/power_test.go

```diff
@@ -581,6 +846,28 @@ func (h *spActorHarness) getClaim(rt *mock.Runtime, a addr.Address) *power.Claim
 	return &out
 }
 
+func (h *spActorHarness) getEnrolledCronTicks(rt *mock.Runtime, epoch abi.ChainEpoch) *adt.Array {
```

Comment:
Consider extracting the `Array` to a Go slice for easier inspection in the test methods

Reply:
This is done.